libs/search-widget/src/components/knowledge-graph/Graph.svelte

```diff
@@ -0,0 +1,186 @@
+<script lang="ts">
```

Comment:
We do not want a direct dependency to d3 because it will increase the size of the widget bundle with unneeded stuff in many use cases.
We should load it from a public CDN (as we do for pdf.js for example).

As I need d3 in the Britinsurance demo, here is a possible implementation but feel free to adapt/re-implement:
https://github.com/nuclia/frontend/blob/ericbrehault/sc-5269/britinsurance-pov/libs/search-widget/src/components/charts/PieChart.svelte#L152

Reply:
Right, I'll do the changes